python/ray/train/_internal/backend_executor.py

```diff
@@ -120,6 +120,13 @@ def start(
         # TODO remove
         if self._trial_info and self._trial_info.driver_ip:
             self.worker_group._move_workers_with_ip_to_front(self._trial_info.driver_ip)
+
+        worker_loc_str = [
+            f"{w.metadata.pid} ({w.metadata.node_ip})"
+            for w in self.worker_group.workers
+        ]
+        logger.info(f"Starting distributed worker processes: {worker_loc_str}")
```

Comment:
Super nit: Since technically this is a list of strings...
```suggestion
        worker_loc_str = [
            f"{w.metadata.pid} ({w.metadata.node_ip})"
            for w in self.worker_group.workers
        ]
        logger.info(f"Starting distributed worker processes: {worker_loc_str}")
```

```suggestion
        worker_locs = [
            f"{w.metadata.pid} ({w.metadata.node_ip})"
            for w in self.worker_group.workers
        ]
        logger.info(f"Starting distributed worker processes: {worker_locs}")
```